.github/workflows/headlamp-plugin-github-workflow.yaml

```diff
@@ -22,15 +20,16 @@ jobs:
     strategy:
       matrix:
         node-version: [18.x]
+        plugin-name: [app-catalog, prometheus]
```

Comment:
Yeah, maybe we can do what @illume was suggesting by running the steps below after navigating through each result of `grep -m1 -lnR @kinvolk/headlamp-plugin ./*/package.json` and running on the resulting paths?